CHANGES.txt

```diff
@@ -2,12 +2,65 @@
 Chaco CHANGELOG
 ===============
 
+Release 4.8.0
+-------------
+
+New features/Improvements
+
+* Refactor DataPrinter to add `_build_text_from_event` method (PR #450)
```

Comment:
Is this changelog thinking of itself as reST, Markdown, or neither? If reST, the backticks should be doubled.

Reply:
I removed the backticks altogether (the rest of the changelog doesn't have any code literals at all). We can make it into an official reST file later on.